### PR TITLE
Add basic options for the chameleon skin

### PR DESCRIPTION
**Pull request by govynnus**
Makes `$egChameleonLayoutFile` and `$egChameleonEnableExternalLinkIcons` configurable.

https://github.com/ProfessionalWiki/chameleon/blob/master/docs/customization.md#layout-of-page-elements
https://github.com/ProfessionalWiki/chameleon/blob/master/docs/customization.md#enable-external-link-icons

### CODE REVIEW

**Comment by govynnus** [comment]
Do you know if there's any way to add dictionary-like options with arbitrary key-value pairs? Or would this require additions to ManageWiki?

That kind of input would be required for the other Chameleon settings, e.g. [$egChameleonExternalStyleVariables](https://github.com/ProfessionalWiki/chameleon/blob/master/docs/customization.md#changing-existing-scss-variables). (I think, unless there's a clever way to do it without).

**Comment by Universal-Omega** [comment]
> Do you know if there's any way to add dictionary-like options with arbitrary key-value pairs? Or would this require additions to ManageWiki?
> 
> That kind of input would be required for the other Chameleon settings, e.g. [$egChameleonExternalStyleVariables](https://github.com/ProfessionalWiki/chameleon/blob/master/docs/customization.md#changing-existing-scss-variables). (I think, unless there's a clever way to do it without).

Something like `list-multi-bool` (IE what `$wgAbuseFilterActions` does in ManageWiki)? That requires a set one, but not arbitrary, and that would require changes to ManageWiki.

File: LocalSettings.php [hunk 1]
```diff
@@ -477,6 +477,9 @@
 		'obeymewiki' => '/srv/mediawiki/w/skins/chameleon/layouts/fixedhead.xml',
```
Comment (Universal-Omega) [inline LocalSettings.php:477]: Wiki overrides should be removed here now. Keeping only default.

File: ManageWikiSettings.php [hunk 2]
```diff
@@ -3548,6 +3548,31 @@
...
+		'overridedefault' => '/srv/mediawiki/w/skins/chameleon/layouts/standard.xml',
+		'section' => 'styling',
+		'help' => 'The layout to use for the Chameleon skin.',
+		'requires' => [],
+	],
+	'egChameleonEnableExternalLinkIcons' => [
+		'name' => 'Chameleon external link icons',
+		'from' => 'chameleon',
+		'type' => 'check',
+		'overridedefault' => 'false',
```
Comment (Universal-Omega) [inline ManageWikiSettings.php:3571]:
```suggestion
		'overridedefault' => false,
```

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiSettings.php (hunk 2 above)
Comment (govynnus) [reply ManageWikiSettings.php]: woops

File: LocalSettings.php (hunk 1 above)
Comment (govynnus) [reply LocalSettings.php]: Done. Is there a way to let those wikis know or set the option for them?

Comment (Universal-Omega) [reply LocalSettings.php]: > Done. Is there a way to let those wikis know or set the option for them?

We can set them once merged.

**Comment by govynnus** [comment]
> Something like `list-multi-bool` (IE what `$wgAbuseFilterActions` does in ManageWiki)? That requires a set one, but not arbitrary, and that would require changes to ManageWiki.

Yeah I think `list-multi-bool` wouldn't quite work, would the project be open to / interested in those changes to ManageWiki?

**Comment by Universal-Omega** [comment]
> > Something like `list-multi-bool` (IE what `$wgAbuseFilterActions` does in ManageWiki)? That requires a set one, but not arbitrary, and that would require changes to ManageWiki.
> 
> Yeah I think `list-multi-bool` wouldn't quite work, would the project be open to / interested in those changes to ManageWiki?

I think if the changes were made, we could implement it yeah.

**Comment by govynnus** [comment]
> I think if the changes were made, we could implement it yeah.

Cool :)

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**